.github/workflows/tests.yml

```diff
@@ -113,34 +113,35 @@ jobs:
         shell: bash
         if: env.GIT_DIFF
 
-  test_apps:
-    runs-on: ubuntu-latest
-    needs: build
-    timeout-minutes: 5
-    steps:
-      - uses: actions/setup-go@v2
-        with:
-          go-version: "^1.15.4"
-      - uses: actions/checkout@v2
-      - uses: technote-space/get-diff-action@v4
-        with:
-          PATTERNS: |
-            **/**.go
-            go.mod
-            go.sum
-      - uses: actions/cache@v2.1.4
-        with:
-          path: ~/go/pkg/mod
-          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-go-
-        if: env.GIT_DIFF
-      - uses: actions/cache@v2.1.4
-        with:
-          path: ~/go/bin
-          key: ${{ runner.os }}-${{ github.sha }}-tm-binary
-        if: env.GIT_DIFF
-      - name: test_apps
-        run: test/app/test.sh
-        shell: bash
-        if: env.GIT_DIFF
+  # TODO: re-enable this test after upgrading to tendermint v0.36.x
+  # test_apps:
+  #   runs-on: ubuntu-latest
+  #   needs: build
+  #   timeout-minutes: 5
+  #   steps:
+  #     - uses: actions/setup-go@v2
+  #       with:
+  #         go-version: "^1.15.4"
+  #     - uses: actions/checkout@v2
+  #     - uses: technote-space/get-diff-action@v4
+  #       with:
+  #         PATTERNS: |
+  #           **/**.go
+  #           go.mod
+  #           go.sum
+  #     - uses: actions/cache@v2.1.4
+  #       with:
+  #         path: ~/go/pkg/mod
+  #         key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+  #         restore-keys: |
+  #           ${{ runner.os }}-go-
+  #       if: env.GIT_DIFF
+  #     - uses: actions/cache@v2.1.4
+  #       with:
+  #         path: ~/go/bin
+  #         key: ${{ runner.os }}-${{ github.sha }}-tm-binary
+  #       if: env.GIT_DIFF
+  #     - name: test_apps
+  #       run: test/app/test.sh
```

Comment:
I think it is ok to disable those even after the upgrade to 0.36 but only if do we have other integration tests that prove that our app still works with core.

Reply:
I agree, we have the e2e test, when tests the new abci++ methods in process with a [minimal app](https://github.com/celestiaorg/celestia-core/blob/314ebc944d8b4f68ed2eb18c9af19d18c78b4df3/test/e2e/app/app.go#L257-L265), and then we also have a cli integration test for the app that [spins up a single validator network and submits a PayForData](https://github.com/celestiaorg/celestia-app/blob/master/x/payment/client/testutil/integration_test.go) in the app that is currently using the new abci++ methods

I'm optimistic that the bug is caused by some silly mistake in the socket server/client, which is ironed out in the official ABCI++ implementation, but if this turns out to not be the case, then I think we should look into this again.